bob-common/src/error.rs

```diff
@@ -168,6 +178,7 @@ impl From<Status> for Error {
                 "PearlChangeState" => Some(Self::pearl_change_state(rest_words(words, length))),
                 "Unauthorized" => Some(Self::unauthorized()),
                 "HolderTemporaryUnavailable" => Some(Self::holder_temporary_unavailable()),
+                _ if code == Code::Unavailable => Some(Self::network()),
```

Comment:
This will never be chosen, because in the case of `Unavailable` error the `name` will be None` on the first `match`